dataservice/util/data_gen/data_generator.py

```diff
@@ -0,0 +1,316 @@
+        for line in reader:
+            self.instrument_model_list.append(line[0])
+
+        self.is_paired_end_list = [True, False]
+        self.library_strand_list = ['Unstranded', 'First_Stranded',
+                                    'Second_Stranded']
+        self.experiment_strategy_list = [
+            'WGS',
+            'WXS',
+            'RNA-Seq',
+            'ChIP-Seq',
+            'miRNA-Seq',
+            'Bisulfite-Seq',
+            'Validation',
+            'Amplicon',
+            'Targeted Sequencing',
+            'Other']
+        self.platform_list = ['Illumina', 'SOLiD', 'LS454', 'Ion Torrent',
+                              'Complete Genomics', 'PacBio', 'Other']
+
+    def _demographics_choices(self):
+        """
+        Provides the choices for filling Demographics entity
+        """
+        self.race_list = [
+            "White",
+            "Black or African American",
+            "Asian",
+            "Native Hawaiian or Other Pacific Islander",
+            "American Indian or Alaska Native",
+            "Other",
+            "Unavailable",
+            "Not Reported",
+            "not allowed to collect"]
+        self.ethnicity_list = [
+            "hispanic or latino",
+            "not hispanic or latino",
+            "Unknown",
+            "not reported",
+            "not allowed to collect"]
+        self.gender_list = ['female', 'male', 'unknown', 'unspecified',
+                            'not reported']
+
+    def _diagnoses_choices(self):
+        """
+        Provides the choices for filling Diagnosis entity
+        """
+        self.max_diagnoses = 35
+        self.min_diagnoses = 0
+        dref_file = open('dataservice/util/data_gen/diagnoses.txt', 'r')
+        reader = csv.reader(dref_file)
+        self.diagnosis_list = []
+        for line in reader:
+            self.diagnosis_list.append(line[0])
+
+    def _genomic_files_choices(self):
+        """
+        Provides the choices for filling Genomic File entity
+        """
+        self.min_gen_files = 0
+        self.max_gen_files = 5
+        self.file_format_list = ['.cram', '.bam', '.vcf']
+
+    def setup(self, config_name):
+        """
+        creates tables in database
+        """
+        self.app = create_app(config_name)
+        self.app_context = self.app.app_context()
+        self.app_context.push()
+        db.create_all()
+
+    def teardown(self):
+        db.session.remove()
+        self.app_context.pop()
+
+    def drop_all(self):
+        db.session.remove()
+        db.drop_all()
+        self.app_context.pop()
+
+    def create_and_publish_all(self):
+
+        # Create participants
+        self._create_participants(self.max_participants)
+        self.teardown()
+
+    def _create_participants(self, total):
+        """
+        creates participants with samples, demographics, and diagnoses
+        """
+        for i in range(total):
+            self.p = Participant(
```

Comment:
p probably doesn't need to be a class property. Same comment as below about assigning samples and other relationships beforehand.